lib/libxdp/README.org

```diff
@@ -8,17 +8,20 @@
 # To export the man page, simply use the org-mode exporter; (require 'ox-man) if
 # it's not available. There's also a Makefile rule to export it.
 
-* libxdp - library for attaching XDP programs
+* libxdp - library for attaching XDP programs and using AF_XDP sockets
 
-This directory contains the files for the =libxdp= library for attaching XDP
-programs to network interfaces. The library is fairly lightweight and relies on
-=libbpf= to do the heavy lifting for processing eBPF object files etc.
+This directory contains the files for the =libxdp= library for
+attaching XDP programs to network interfaces and using AF_XDP
+sockets. The library is fairly lightweight and relies on =libbpf= to
+do the heavy lifting for processing eBPF object files etc.
 
-The primary feature that =libxdp= provides on top of =libbpf= is the ability to
-load multiple XDP programs in sequence on a single network device (which is not
-natively supported by the kernel). This support relies on the =freplace=
-functionality in the kernel, which makes it possible to attach an eBPF program
-as a replacement for a global function in another (already loaded) eBPF program.
+One of the two primary featurea that =libxdp= provides on top of
+=libbpf= is the ability to load multiple XDP programs in sequence on a
+single network device (which is not natively supported by the
+kernel). This support relies on the =freplace= functionality in the
+kernel, which makes it possible to attach an eBPF program as a
+replacement for a global function in another (already loaded) eBPF
+program.
```

Comment:
Also, since this paragraph starts with 'one of the two features', it seems odd that the other one is not mentioned at all; so maybe fix that? :)